scripts/client.js

```diff
@@ -0,0 +1,25 @@
+import KintoClient from "kinto-client";
+
+
+let client;
```

Comment:
I would rather create a stateless client that we can create from the store.
I would then pass the serverInfo from the state if I catched them already.

It don't know if it makes sense.